lib/specinfra/command/freebsd/v6/service.rb

```diff
@@ -0,0 +1,7 @@
+class Specinfra::Command::Freebsd::V6::Service < Specinfra::Command::Base::Service
+  class << self
+    def check_is_enabled(service, level=3)
+      "service -e | grep -- /#{escape(service)}$"
```

Comment:
Fix `/` to `^` ?

Reply:
`/` was intentional, the output of the `service -e` command looks like this:

```
$ service -e
/etc/rc.d/newsyslog
/etc/rc.d/syslogd
/etc/rc.d/dmesg
/usr/local/etc/rc.d/openntpd
/usr/local/etc/rc.d/smartd
/etc/rc.d/sshd
/etc/rc.d/sendmail
/etc/rc.d/cron
```